structure.Rmd

```diff
@@ -330,8 +330,8 @@ Here's a similar look on macOS (but your results may vary):
 ```{r, eval = FALSE}
 # on macOS
 .libPaths()
-#> [1] "/Users/jenny/Library/R/3.6/library"
-#> [2] "/Library/Frameworks/R.framework/Versions/3.6/Resources/library"
+#> [1] "/Users/jenny/Library/R/arm64/4.2/library"
+#> [2] "/Library/Frameworks/R.framework/Versions/4.2-arm64/Resources/library"
```

Comment:
I assume this is based on what you see on your machine. I'm still on Intel, so `.libPaths()` looks different for me now, both vs. what was originally here _and_ different from what you're proposing. But I think your version is more forward-looking.

``` r
.libPaths()
#> [1] "/Users/jenny/Library/R/x86_64/4.2/library"                     
#> [2] "/Library/Frameworks/R.framework/Versions/4.2/Resources/library"
```

<sup>Created on 2023-02-10 with [reprex v2.0.2.9000](https://reprex.tidyverse.org)</sup>

Reply:
Yeah, the architecture is now explicitly part of the path in `R_LIBS_USER` according to [`?libPaths`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/libPaths.html)